routers/web/repo/issue.go

```diff
@@ -2078,6 +2078,42 @@ func ViewIssue(ctx *context.Context) {
 		return user_service.CanBlockUser(ctx, ctx.Doer, blocker, blockee)
 	}
 
+	if ctx.IsSigned {
+		forkedRepos, err := repo_model.GetForksByUserAndOrgs(ctx, ctx.Doer, ctx.Repo.Repository)
+		if err != nil {
+			ctx.ServerError("GetForksByUserAndOrgs", err)
+			return
+		}
+		allowedRepos := make([]*repo_model.Repository, 0, len(forkedRepos)+1)
+		for _, repo := range append(forkedRepos, ctx.Repo.Repository) {
+			perm, err := access_model.GetUserRepoPermission(ctx, repo, ctx.Doer)
+			if err != nil {
+				ctx.ServerError("GetUserRepoPermission", err)
+				return
+			}
+			if perm.CanWrite(unit.TypeCode) {
+				allowedRepos = append(allowedRepos, repo)
+			}
+		}
+
+		ctx.Data["AllowedRepos"] = allowedRepos
+
+		devLinks, err := issue_service.FindIssueDevLinksByIssue(ctx, issue)
+		if err != nil {
+			ctx.ServerError("FindIssueDevLinksByIssue", err)
+			return
+		}
+		ctx.Data["DevLinks"] = devLinks
+		for _, link := range devLinks {
+			if link.LinkType == issues_model.IssueDevLinkTypePullRequest {
+				if !(link.PullRequest.Issue.IsClosed && !link.PullRequest.HasMerged) {
```

Comment:
This can be merged

Reply:
How?

Reply:
if link.LinkType == issues_model.IssueDevLinkTypePullRequest && !link.PullRequest.Issue.IsClosed && !link.PullRequest.HasMerged

Do you mean `not closed and not merged PRs`? The `!` seems wrong. The first one is outside of the `(xxxx)`

Reply:
No. This means opened pull request or closed but merged pull request. Only closed and not merged pull request should not be considered as a maybe-fixed pull request.

Reply:
66681c33ffd42f636528a40a0b0c3fcd1a37aba2

Reply:
Does GitHub also have `May be fixed`?
PR is not merged, so there's no code changes, so why it can be `May be fixed`? There's no fix.